pkg/reconciler/buildrun/resources/taskrun.go

```diff
@@ -248,42 +229,10 @@ func GenerateTaskRun(
 		Spec: v1beta1.TaskRunSpec{
 			ServiceAccountName: serviceAccountName,
 			TaskSpec:           taskSpec,
-			Resources: &v1beta1.TaskRunResources{
-				Inputs: []v1beta1.TaskResourceBinding{
-					{
-						PipelineResourceBinding: v1beta1.PipelineResourceBinding{
-							Name: inputSourceResourceName,
-							ResourceSpec: &taskv1.PipelineResourceSpec{
-								Type: taskv1.PipelineResourceTypeGit,
-								Params: []taskv1.ResourceParam{
-									{
-										Name:  inputGitSourceURL,
-										Value: build.Spec.Source.URL,
-									},
-									{
-										Name:  inputGitSourceRevision,
-										Value: revision,
-									},
-								},
-							},
-						},
-					},
-				},
-				Outputs: []v1beta1.TaskResourceBinding{
-					{
-						PipelineResourceBinding: v1beta1.PipelineResourceBinding{
-							Name: outputImageResourceName,
-							ResourceSpec: &taskv1.PipelineResourceSpec{
-								Type: taskv1.PipelineResourceTypeImage,
-								Params: []taskv1.ResourceParam{
-									{
-										Name:  outputImageResourceURL,
-										Value: image,
-									},
-								},
-							},
-						},
-					},
+			Workspaces: []v1beta1.WorkspaceBinding{
+				{
+					Name:     workspaceSource,
```

Comment:
Nice. I think the intention of this one is to make the source code available for further steps in the TaskRun, under the `/workspace/source` path?  Could you add a comment here explaining that pls.

Reply:
Done.